RecoHI/HiTracking/python/HISelectedVertex_cfi.py

```diff
@@ -7,11 +7,57 @@
 )
```

Comment:
this file should be renamed to a cff.py
the name can be bestHiVertexSequences_cff.py or something similar
"HISelectedVertex" does not represent the content anymore.